drivers/SmartThings/zigbee-switch/src/aqara-light/init.lua

```diff
@@ -32,13 +33,15 @@ local function do_refresh(self, device)
   device:send(ColorControl.attributes.ColorTemperatureMireds:read(device))
 end
 
-local function do_configure(self, device)
-  device:configure()
+local function device_added(driver, device, event)
   device:send(cluster_base.write_manufacturer_specific_attribute(device,
     PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 1)) -- private
+end
+
+local function do_configure(self, device)
+  device:configure()
 
-  device:send(Level.attributes.OnTransitionTime:write(device, 0))
-  device:send(Level.attributes.OffTransitionTime:write(device, 0))
```

Comment:
The `Aqara` devices that have already had this applied will not be fixed with this change, only new installs. Is that okay?

If you want to reset to default, I believe you'd have to send `0xFFFF`.

Reply:
You're right. It's intended.

The original driver's intention was to set the On/Off Transition Time to 0 during the onboarding process in the doConfigure handler, allowing users to adjust it later using the stse.lightFadeIn/OutTimeInSec preferences (with a default value of 0). 
However, in case of re-onboarding scenario, there was a mismatch issue caused by the fact that when re-onboarding occurred, the previously set preference values remained in place at cloud, while the doConfigure function forced them back to 0. 
To resolve this, I made a modification where, during new onboarding, this patch synchronizes the preference values (defaulting to 0). In contrast, during re-onboarding, instead of forcing a change to 0, this patch resynchronizes the values previously set by the user.

Reply:
Only one of the models on this sub-driver actually has the preference, though.

As your test demonstrates, one of them no longer gets this value set, and it's not a user preference either.

Reply:
> Only one of the models on this sub-driver actually has the preference, though.
> 
> As your test demonstrates, one of them no longer gets this value set, and it's not a user preference either.

Hi @Hongming6 

Could you help confirm these changes and common ? 

Thanks.

Reply:
Hi@Kwang-Hui @greens @liuyd96 
Since the Aqara LED Bulb T1 doesn't have this preference, we  think it's not necessary to set the Aqara LED Bulb T1's On/OffTransitionTime to 0, and we can use the default fade time of the device itself.